controllers/customers_controller.js

```diff
@@ -0,0 +1,63 @@
+var Customer = require("../models/customer");
+
+var CustomersController = {
+
+  getCustomers: function(req, res, next) {
+    // giving a callback function to handle error or render view
```

Comment:
Remember to remove any unnecessary comments before submitting your final PR.  They are super helpful during development, though, huh? :)